frontend/src/components/static_pages/about.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import "./about.css";
-import SingleCarousel from "../SingleCarousel";
+import CarouselComponent from "../home/carousel";
```

Comment:
Name choice - I'd recommend renaming to remove `Component` in name. 

Using variable names such as `CarouselComponent` or `nameString` is referred to as Hungarian Notation, where you include the type in the name. It used to be popular but is generally not used anymore.

Reply:
Thank you for the advice. It looks like the name `Carousel` is a component in React. Perhaps that's why the original developer named it `CarouselComponent`. There are multiple places where it is referenced. I changed all of these references to `MainCarousel`.